lib/prometheus/business_collector.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+unless defined? Rails
+  require File.expand_path('../../config/environment', __dir__)
+end
+
+# rubocop:disable Metrics/MethodLength
+# rubocop:disable Metrics/AbcSize
+# Collects stats relevent for the business from the database
+# when /metrics is called. This runs directly on prometheus_exporter.
+class BusinessCollector < PrometheusExporter::Server::TypeCollector
+  def initialize
+    @total_accounts = PrometheusExporter::Metric::Gauge.new(
```

Comment:
These are all `Gauge` vs `Counter` as theoretically they can all go up and down. We don't remove unused accounts, but we could.